test/generators/go/GoConstrainer.spec.ts

```diff
@@ -184,3 +184,167 @@ describe('GoConstrainer', () => {
     });
   });
 });
+
+describe('nullable & requried properties', () => {
```

Comment:
For simplicity, I would recommend merging this grouping of tests with the regular tests, i.e. you have 4 tests under `describe('String', () => {` to keep it simple and a single place to look for the tests for specific type.